proposals/csharp-12.0/ref-readonly-parameters.md

```diff
@@ -95,10 +99,25 @@ By-value overloads will be preferred over `ref readonly` overloads in case there
 #### Method conversions
 [method-conversions]: #method-conversions
 
-Similarly, for the purpose of anonymous function [[§10.7](https://github.com/dotnet/csharpstandard/blob/47912d4fdae2bb8c3750e6485bdc6509560ec6bf/standard/conversions.md#107-anonymous-function-conversions)] and method group [[§10.8](https://github.com/dotnet/csharpstandard/blob/47912d4fdae2bb8c3750e6485bdc6509560ec6bf/standard/conversions.md#108-method-group-conversions)] conversions, these modifiers are considered compatible (but the conversion results in a warning):
+Similarly, for the purpose of anonymous function [[§10.7](https://github.com/dotnet/csharpstandard/blob/47912d4fdae2bb8c3750e6485bdc6509560ec6bf/standard/conversions.md#107-anonymous-function-conversions)] and method group [[§10.8](https://github.com/dotnet/csharpstandard/blob/47912d4fdae2bb8c3750e6485bdc6509560ec6bf/standard/conversions.md#108-method-group-conversions)] conversions, these modifiers are considered compatible
+(but any allowed conversion between different modifiers results in a warning):
+
+- `ref readonly` can be interchanged with `in` and converted to `ref` modifier (but `ref` cannot be converted to `ref readonly`),
```

Comment:
I find this wording confusing. We are not converting modifiers. Interchangeability is also a confusing term in this context, I think.  Are we trying to say that a `ref readonly` parameter of the target method is allowed to match `in` or `ref` parameter of the delegate? Consider using this wording then.